scripts/qesap/test/e2e/test.sh

```diff
@@ -288,3 +288,24 @@ ansible_logs_number=$(find . -type f -name "ansible.*.log.txt" | wc -l)
 # 3 playbooks plus a log file for the initial ansible (not ansible-playbook) call 
 [[ $ansible_logs_number -eq 4 ]] || test_die "ansible .log.txt are not 4 files but has ${ansible_logs_number}"
 rm ansible.*.log.txt
+
+test_step "[test_4.yaml] Run Ansible with --junit"
+find . -type f -name "sambuconero*.xml" -delete || echo "Nothing to delete"
+qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml ansible --junit . || test_die "test_4.yaml fail on ansible"
+junit_logs_number=$(find . -type f -name "sambuconero*.xml" | wc -l)
+[[ $junit_logs_number -eq 1 ]] || test_die "ansible JUNIT reports should be 1 files but are ${junit_logs_number}"
+find . -type f -name "sambuconero*.xml" -delete
+rm ansible.*.log.txt
+
+test_step "[test_4.yaml] Run Ansible with --profile"
```

Comment:
this test can be improved. It is fine but it does not check is profile info are in the output or not. Please incorporate test implementation from https://github.com/SUSE/qe-sap-deployment/pull/254/files#diff-211ace378c4b9937437dae88a2a8e6f50efb2315618a3f77a124410274d8f36dR305

It is also fine you just look at it and provide your own implementation

Reply:
Done. Thanks for the comments.